app/lib/actions/katello/host/update.rb

```diff
@@ -18,6 +18,7 @@ def plan(host, consumer_params = nil)
               plan_action(::Actions::Candlepin::Consumer::Update, host.subscription_facet.uuid, consumer_params)
             elsif host.subscription_facet
               consumer_params = host.subscription_facet.consumer_attributes
+              plan_action(::Actions::Candlepin::Consumer::Update, host.subscription_facet.uuid, consumer_params)
```

Comment:
Maybe another layer of conditional logic to DRY up this plan_action based upon whether `host.subscription_facet` is true?